acceptance/lib/puppet/acceptance/service_utils.rb

```diff
@@ -0,0 +1,21 @@
+module Puppet
+  module Acceptance
+    module ServiceUtils
+
+      # Return whether a host supports the Systemd provider.
+      # @param host [String] hostname
+      # @return [Boolean] whether the systemd provider is supported.
+      def supports_systemd? (host)
```

Comment:
This is going to require maintenance. What if we did the same thing the puppet code does in the provider: 
https://github.com/puppetlabs/puppet/blob/2871aaff1a5a5e1dbbeb0e485801dc4467caab6e/lib/puppet/provider/service/systemd.rb#L8?